probe-rs/src/probe/jlink/error.rs

```diff
@@ -22,8 +22,10 @@ pub enum JlinkError {
     /// advertised as optional *capability* bits. This error occurs when the capability bit for an
     /// operation isn't set when that operation is attempted.
     ///
-    /// Capabilities can be read by calling [`JLink::capabilities`], which returns a
-    /// [`Capabilities`] struct.
+    /// Capabilities can be read by calling [`super::JLink::capabilities()`][cap], which returns a
+    /// [`Capabilities`][super::Capabilities] struct.
+    ///
+    /// [cap]: super::JLink::capabilities()
```

Comment:
Isn't this redundant?

Reply:
If you mean defering the link at the end of the block, it does not show in the rendered doc as a second link but helps making the line in the code within reasonable margin.

Reply:
I mean I think it's redundant because ``[`super::JLink::capabilities()`]`` should be a valid intra-doc link as it is, and the late definition should be unnecessary.

Reply:
Oh, that’s a good point 👍